sysutils/scp-backup/src/opnsense/scripts/OPNsense/ScpBackup/ScpBackup.php

```diff
@@ -39,12 +39,14 @@
 $config = Config::getInstance()->object();
 $scpBackup = $config->OPNsense->ScpBackup;
 
+define('TMP_PRIVATE_KEY', 'tmp-private-key-');
+define('TMP_HOST_KEY', "tmp-host-key-");
```

Comment:
use /tmp/os_known_hosts (global file)

Reply:
See above. I perfer my approach for doing this.